src/data/reference-data/ReferenceDataImporter.js

```diff
@@ -27,7 +28,12 @@ export const ReferenceDataImporter = ({ onSuccess, onDismiss, namespace, name })
             try {
               await Ajax()
                 .Workspaces.workspace(namespace, name)
-                .shallowMergeNewAttributes(_.mapKeys((k) => `referenceData_${selectedReference}_${k}`, ReferenceData[selectedReference]));
+                .shallowMergeNewAttributes(
+                  _.mapKeys(
+                    (k) => `referenceData_${extractReferenceName(selectedReference)}_${k}`,
```

Comment:
This is the type of thing I'd like to avoid. We should never have to go backwards from a reference label to a reference ID. All state should be based on the reference ID and then there can be a one-way mapping from ID to label.